python/tvm/relay/transform/memory_alloc.py

```diff
@@ -26,8 +26,13 @@
 from ... import DataType, register_func
 from .. import ty, expr
 from ..backend import compile_engine
+from ..op.memory import alloc_storage, alloc_tensor
 
 
+def alloc_tensor(storage, shape, dtype='float32', assert_shape=None):
+    offset = expr.const(0, dtype="int64")
```

Comment:
so that offset is actually always zero? then we can just directly use 0 when allocating tensor?

Reply:
We initially allocate 0, then we later change the offset to be non-zero.